test/libsolidity/semanticTests/various/selfdestruct_post_cancun.sol

```diff
@@ -0,0 +1,51 @@
+contract C {
+    event Terminated();
+
+    constructor() payable {}
+
+    function terminate() external {
+        emit Terminated();
+        // NOTE: The contract `c` should still exists in the test below,
+        // since the call to the selfdestruct method was done in a tx that is
+        // not the same tx that the contract was created.
+        // However, it should send all Ether in `c` to the beneficiary.
+        selfdestruct(payable(msg.sender));
+        // Execution stops here, so the second one is not executed.
+        selfdestruct(payable(msg.sender));
```

Comment:
Why include it if can't see if it happens or not?

Reply:
It was included in the original test. And I think the idea is to ensure that the event after the second destruct is never emitted. But yeah, we could remove those I guess.

Reply:
Removed.